hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/WatermarkPolicy.java

```diff
@@ -98,4 +90,27 @@ public long getCurrentWatermark() {
             }
         };
     }
+
+    /**
+     * Maintains a watermark that lags behind the real time by the given
+     * amount. Doesn't consider the event timestamp at all.
+     *
+     * @param lag the desired difference between the top observed timestamp
+     *            and the watermark
+     */
+    @Nonnull
```

Comment:
missing `@since`

Reply:
It's core API, do we care? It can also be gone...